api/types.proto

```diff
@@ -27,6 +27,9 @@ message Resources {
 
 	// Amount of memory in bytes.
 	int64 memory_bytes = 2;
+
+	// Container pid limit
+	int64 pids_limit = 3;
```

Comment:
I wonder if this should really be considered a resource limit. The other items here are resources shared among containers on the host, but PIDs are scoped to the container. Maybe this is indeed the right place to put it, but I want to make sure.

ping @stevvooe @justincormack